### PR TITLE
Open uploaded report in new tab

### PR DESCRIPTION
**Pull request by MarkvdVorst**
Instead of opening the uploaded file in the debug tab, it is now shown in its own seperate tab. This is mainly to give a better view of the uploaded report while not uploading it to the ladybug storage. This also removes some buttons that would not work with uploaded reports like the copy button.

Changed the Cypress test for the file upload.

### CODE REVIEW

File: src/app/report/report.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="report-tab row h-100">
   <div class="resizable-horizontal w-25 p-2 border-right h-100">
-    <jqxTree #treeReference (onItemClick)="selectReport($event)" [auto-create]="false"></jqxTree>
+    <jqxTree #treeReference (onItemClick)="selectReport($event)" [auto-create]="false" data-cy-debug="jqxTree"></jqxTree>
```
Comment (MatthijsSmets) [inline src/app/report/report.component.html:3]: Seeing as this tree is in the report component and not the debug, it should be:
```suggestion
    <jqxTree #treeReference (onItemClick)="selectReport($event)" [auto-create]="false" data-cy-report="jqxTree"></jqxTree>
```

File: cypress/e2e/debug_fileupload.cy.js [hunk 2]
```diff
@@ -17,7 +14,6 @@ describe('Debug file upload', () => {
           fileName: 'testRerun.ttr',
         });
       });
-
-    cy.checkFileTreeLength(1);
+    cy.get('[data-cy-debug="jqxTree"]').find('li').should('have.length', 3);
```
Comment (MatthijsSmets) [inline cypress/e2e/debug_fileupload.cy.js:17]:
```suggestion
    cy.get('[data-cy-report="jqxTree"]').find('li').should('have.length', 3);
```

**Review by MatthijsSmets — CHANGES_REQUESTED** [review]

File: src/app/report/report.component.html (hunk 1 above)
Comment (MarkvdVorst) [reply src/app/report/report.component.html]: added the change

File: cypress/e2e/debug_fileupload.cy.js (hunk 2 above)
Comment (MarkvdVorst) [reply cypress/e2e/debug_fileupload.cy.js]: added the change

**Review by MatthijsSmets — APPROVED** [review]

**Pull request merged**